services/azure-storage/src/lib.rs

```diff
@@ -19,3 +20,34 @@ pub use credential::Credential;
 
 mod loader;
 pub use loader::Loader;
+
+/// The Azure Storage service that a configuration or credential is used with.
+#[derive(PartialEq)]
+pub enum Service {
```

Comment:
Do you think it's good idea we just accept plain string?

Reply:
I actually don't want to expose such API. Maybe a public constant would be sufficient?

Reply:
That's fair! Will make the change 👍